director/director_api.go

```diff
@@ -76,29 +76,24 @@ func CreateDirectorSDToken() (string, error) {
 	if directorURL == "" {
```

Comment:
For this one, since the only use for the function is to create a token, let's remove this function and put your code to where the function was referred (which should be only one place)